src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,35 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        File file = new File(fromFileName);
+        int supply = 0;
+        int buy = 0;
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(toFileName))) {
+            List<String> strings = Files.readAllLines(file.toPath());
 
+            for (String part : strings) {
+                String[] parts = part.split(",");
```

Comment:
Prefer using 'split()' over 'substring()', but since you're already using 'split()' correctly, ensure that you handle potential issues like incorrect data format.